pkgs/top-level/all-packages.nix

```diff
@@ -3014,7 +3014,7 @@ with pkgs;
 
   swaysome = callPackage ../tools/wayland/swaysome { };
 
-  swaytools = callPackage ../tools/wayland/swaytools { };
+  swaytools = python3Packages.callPackage ../tools/wayland/swaytools { };
```

Comment:
Adding this so I can use buildPythonApplication directly, without needing to use python3Packages.buildPythonApplication.